R/pkg/NAMESPACE

```diff
@@ -357,6 +357,7 @@ exportMethods("%<=>%",
               "to_utc_timestamp",
               "translate",
               "trim",
+              "trunc",
```

Comment:
It is part of the internally S4 methods and there is already generics.
This is similar to math functions like `abs`.
https://stat.ethz.ch/R-manual/R-devel/library/base/html/groupGeneric.html